### PR TITLE
[#18] Add a documentation page for fieldRoles

### LINKED ISSUE #18: Reco and Simi are missing info about fieldRoles
Currently there is no mention of fieldRoles in the documentation about similar products and recommendations. If fieldRoles aren't set when simi/reco response is received, prices aren't localized properly.

### PR DESCRIPTION
**Pull request by damrath**
Fixes #18

### CODE REVIEW

**Comment by damrath** [comment]
#22 will create a merge conflict due to the renamed guides/documentation directory. I will resolve the conflict and re-issue this PR when this is solved

**Comment by damrath** [comment]
Merge conflict resolved.

File: markdown/en/documentation/field-roles.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+product's price in your product database. During set-up you assign the
+role `price` to your field `prix` in the FACT-Finder-UI. In order to use the [Currency Attributes](https://web-components.fact-finder.de/documentation/currency-guide) on your product detail page, you have to map `price` to
+`prix` in the `factfinder.communication.fieldRoles` property in your
+front-end.
+
+**NOTICE:** The `fieldRoles` are configured in the FACT-Finder-UI during
+the set-up process. It is essential the roles you define in the browser
+match the ones defined in FACT-Finder.
+
+#### How to define `fieldRoles`
```
Comment (KrzysztofKozlowskiOBSS) [inline markdown/en/documentation/field-roles.md]: "... on the WebComponents side"
or similar sentence to clearly distinguish between FF UI field roles configuration and FF WC

File: markdown/en/documentation/field-roles.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+variants are used
+
+**IMPORTANT:** During set-up of a channel in the FACT-Finder-UI you have
+to assign each of the `fieldRoles` listed above to one field/column in
+your product database.
+
+This means in order for FACT-Finder and Web Components to process your
+data properly you also need to provide a mapping of `fieldRoles` to
+`field names` in your front-end before you request data via Web Components
+from any API except the search API.
```
Comment (KrzysztofKozlowskiOBSS) [inline markdown/en/documentation/field-roles.md]: Maybe it's worth pointing out exact situations when customers have to provide the mapping on the frontend side.

Comment (toarm) [reply markdown/en/documentation/field-roles.md]: `ff-recommendation`
`ff-similar-products`
`<ff-communication currency-code ...>`
`ff-suggest`
`ff-checkout-tracking`
`ff-tag-cloud` -> which is an awesome element btw. but no one is using it x/
`ff-campaign-pushed-products` (Tracking) loaded via `ff-campaign-landing-page`, `ff-campaign-products`, `ff-campiang-shopping-cart`

If I missed something we can add it later as soon as we get an "bug" issue

**Comment by toarm** [comment]
https://documentation-6bbfc.firebaseapp.com/documentation/field-roles

**Comment by KrzysztofKozlowskiOBSS** [comment]
@toarm the link seems to be broken, was the deployment successful?

**Pull request merged**